### ISSUE TITLE
Main cdc sync not support delete op

### ISSUE THREAD

**Issue by xlfjcg** [StarRocks/starrocks-connector-for-apache-flink#129]
flink cdc to sr, when mysql delete, sr did not delete.

**Comment by huoyue-git** [comment]
sql
sr
```
 CREATE TABLE `primary_table_decimal_v3_with_null_flink_sink` ( k0 int, v1 date, v2 datetime, v3 char(20), v4 varchar(20), v5 string, v6 boolean, v7 tinyint , v8 smallint , v9 int , v10 bigint , v11 largeint , v12 float , v13 double , v14 decimal(27,9) , v15 decimal32(9,5), v16 decimal64(18,10), v17 decimal128(38,18) ) ENGINE=OLAP primary KEY(k0) COMMENT "OLAP" DISTRIBUTED BY HASH(k0) BUCKETS 3 PROPERTIES ( "replication_num" = "3" );
  CREATE TABLE `primary_table_decimal_v3_with_null_flink` ( k0 int, v1 date, v2 datetime, v3 char(20), v4 varchar(20), v5 string, v6 boolean, v7 tinyint , v8 smallint , v9 int , v10 bigint , v11 largeint , v12 float , v13 double , v14 decimal(27,9) , v15 decimal32(9,5), v16 decimal64(18,10), v17 decimal128(38,18) ) ENGINE=OLAP primary KEY(k0) COMMENT "OLAP" DISTRIBUTED BY HASH(k0) BUCKETS 3 PROPERTIES ( "replication_num" = "3" );
  insert into primary_table_decimal_v3_with_null_flink values  (1,'9999-12-31','9999-12-31 23:59:59','beijingaergertte','haidiansdvgerwwge','tcafvergtrwhtwrht','1','-128','-32768','-2147483648','-9223372036854775808','-170141183460469231731687303715884105728','-3.115','-3.14159','111111111111111111.111111111','1111.11111','11111111.1111111111','11111111111111111111.111111111111111111'),(2,'0001-01-01','0001-01-01 00:00:01','beijing','haidian','asfarewgeragergre','0','127','32767','2147483647','9223372036854775807','170141183460469231731687303715884105727','-3.115','-3.14159','-111111111111111111.111111111','-1111.11111','-11111111.1111111111','-11111111111111111111.111111111111111111'),(3,'2020-06-23','2020-06-23 00:00:04','','','','1','-124','-32764','-2147483644','-9223372036854775804','-18446744073709551612','-2.7','-3.1','12345.123456789','123.123','1234.12345','123456.123456789'),(4,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null);
```
flink
```
 CREATE TABLE `primary_table_decimal_v3_with_null_sink` (
k0 int, v1 date, v2 timestamp, v3 char(20), v4 varchar(20), v5 string, v6 boolean, v7 tinyint , v8 smallint , v9 int , v10 bigint , v11 string, v12 float , v13 double , v14 decimal(27,9) ,  v15 decimal(9,5), v16  decimal(18,10), v17  decimal(38,18), primary key(k0) NOT ENFORCED
      ) with (
         'connector' = 'starrocks',
         'load-url' = 'xxx.xx.xx.xxx:xxxx',
         'jdbc-url' = 'jdbc:mysql://xxx.xx.xx.xxx:xxxx',
         'username' = 'root',
         'password' = '',
         'database-name' = 'test_flink_connector_sink_from_sr',
         'table-name' = 'primary_table_decimal_v3_with_null_flink_sink'
      );


CREATE TABLE `primary_table_decimal_v3_with_null_flink` (
k0 int, v1 date, v2 timestamp, v3 char(20), v4 varchar(20), v5 string, v6 boolean, v7 tinyint , v8 smallint , v9 int , v10 bigint , v11 string, v12 float , v13 double , v14 decimal(27,9) ,  v15 decimal(9,5), v16  decimal(18,10), v17  decimal(38,18)
        ) with (
           'connector' = 'starrocks',
           'scan-url' = 'xxx.xx.xx.xxx:xxxx',
           'jdbc-url' = 'jdbc:mysql://xxx.xx.xx.xxx:xxxx',
           'username' = 'root',
           'password' = '',
           'database-name' = 'test_flink_connector_sink_from_sr',
           'table-name' = 'primary_table_decimal_v3_with_null_flink',
           'scan.connect.timeout-ms' = '60000',
           'scan.params.query-timeout-s' = '6000'
        );

insert into primary_table_decimal_v3_with_null_sink select * from primary_table_decimal_v3_with_null_flink;
```

**Comment by adhasahar97** [comment]
Hi. Can we have this released?

**Comment by banmoy** [comment]
@adhasahar97 We have fixed this problem in release [1.2.4](https://github.com/StarRocks/starrocks-connector-for-apache-flink/releases/tag/v1.2.4).

**Comment by adhasahar97** [comment]
Ah I see. So it's not another issue. I'll open a new issue.
